src/coreclr/jit/hwintrinsiclistarm64sve.h

```diff
@@ -318,6 +318,8 @@ HARDWARE_INTRINSIC(Sve2,          AbsoluteDifferenceAddWideningLower,
 HARDWARE_INTRINSIC(Sve2,          AbsoluteDifferenceAddWideningUpper,                               -1,      3,     {INS_invalid,        INS_invalid,        INS_sve_sabalt,     INS_sve_uabalt,     INS_sve_sabalt,     INS_sve_uabalt,     INS_sve_sabalt,     INS_sve_uabalt,     INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_BaseTypeFromFirstArg|HW_Flag_HasRMWSemantics)
 HARDWARE_INTRINSIC(Sve2,          AbsoluteDifferenceWideningLower,                                  -1,      2,     {INS_invalid,        INS_invalid,        INS_sve_sabdlb,     INS_sve_uabdlb,     INS_sve_sabdlb,     INS_sve_uabdlb,     INS_sve_sabdlb,     INS_sve_uabdlb,     INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable)
 HARDWARE_INTRINSIC(Sve2,          AbsoluteDifferenceWideningUpper,                                  -1,      2,     {INS_invalid,        INS_invalid,        INS_sve_sabdlt,     INS_sve_uabdlt,     INS_sve_sabdlt,     INS_sve_uabdlt,     INS_sve_sabdlt,     INS_sve_uabdlt,     INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable)
+HARDWARE_INTRINSIC(Sve2,          AddCarryWideningLower,                                            -1,      3,     {INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_invalid,        INS_sve_adclb,      INS_invalid,        INS_sve_adclb,      INS_invalid,        INS_invalid},     HW_Category_SIMD,                  HW_Flag_Scalable|HW_Flag_HasRMWSemantics)
```

Comment:
RMW semantics for this instruction is on 3rd operand 

```
ADCLB <Zda>.<T>, <Zn>.<T>, <Zm>.<T>

<Zda> 
Is the name of the third source and destination scalable vector register, encoded in the "Zda" field.
```

You will need handling of that in lsra as well as in codegen. So this needs `HW_Flag_SpecialCodeGen` as well.